staxing/assignment.py

```diff
@@ -389,7 +390,7 @@ def open_chapter_list(self, driver, chapter):
             By.XPATH,
             '//h2[contains(@data-chapter-section,"%s")]/a' % chapter
         )
-        if not bool(data_chapter.get_attribute('aria_expanded')):
+        if (data_chapter.get_attribute('aria-expanded')) == 'false':
```

Comment:
bool() was returning True for the input 'true' and 'false'. I can add in the helper though